src/utils/duration-humanizer.ts

```diff
@@ -0,0 +1,40 @@
+import * as humanizeDuration from 'humanize-duration';
+
+const getTimeInText = (seconds: number, uiConfig: any, round: boolean = false): string | undefined => {
+  try {
+    const durationHumanizer = getDurationHumanizer(uiConfig);
+    if (durationHumanizer) {
+      try {
+        return seconds ? durationHumanizer(seconds * 1000, {round: round}) : '0';
+      } catch (e: any) {
+        return `${seconds}`;
+      }
+    }
+  } catch (e: any) {
+    // eslint-disable-next-line no-console
+    console.log(e);
```

Comment:
If logging is necessary here, use logger, not console

Reply:
done